### ISSUE TITLE
Capital A displaying wrongly in Google Docs

### ISSUE THREAD

**Issue by NachoGuijarro** [JulietaUla/Montserrat#126]
Hello!
I love this typeface. :)
Is there anybody else experiencing this rendering issue with the crossbar of the capital A in Google docs?

Thanks!

<img width="1300" alt="CapitalAGoogleDocs" src="https://user-images.githubusercontent.com/99727526/154046058-1c94da8d-d3a4-4872-a2b9-1438ca84c8df.png">

**Comment by tstewart-klaudhaus** [comment]
Yes. I see the same issue with the font when installed from NPM with @fontsource/montserrat@^4.5.11. It is particularly noticeable at higher weights but once noticed it can be observed at all sizes and weights.

**Comment by tstewart-klaudhaus** [comment]
Found a solution for my situation (seems to be Mac specific), use the "variable" version of the font as per:

https://github.com/fontsource/fontsource/issues/243

Not sure how you'd do that in Google Docs though?

**Comment by EncyclopediaRed** [comment]
I also just discovered this was happening on one of our machines. Admittedly, it's on an older machine, but it was working up until recently. The heavier weight show almost like a knockout where the crossbar on the A touches the angled arms. The lighter weights show a thinner and thinner crossbar as the weight goes down until there's nothing.

Mac Mini running High Sierra (10.13.6)
Chrome (107), Firefox (106), Safari (13.1.2)

![Screenshot](https://user-images.githubusercontent.com/2585496/198108917-51c09cfd-144e-4037-a107-1829a39acebf.jpg)

**Comment by juandelperal** [comment]
Hi @NachoGuijarro @EncyclopediaRed 

Thanks for reporting this.
I tried the link and the browsers you mentioned, and it looks fine now
Is this issue still happening in your side?

<img width="432" alt="image" src="https://github.com/JulietaUla/Montserrat/assets/2414972/09756a3c-f34c-4a52-b732-cf083088f960">
Mac running Big Sur (11.6)
